Storage/SyncQueue.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  Commands.swift
+//  Client
+//
+//  Created by Emily Toop on 6/29/15.
+//  Copyright (c) 2015 Mozilla. All rights reserved.
+//
+
+import Shared
+
+public struct SyncCommand: Equatable {
+    public let guid: GUID
+    public let modified: Timestamp
+
+    public let title: String?
+    public let url: String?
```

Comment:
This can be simplified a lot. And you should really take a look at `ClientsSynchronizer`, specifically the `Command` protocol. The signature of `fromName:args:` is what a command really looks like, and `DisplayURICommand` is what you're *sending* here.